elide-standalone/src/main/java/com/yahoo/elide/standalone/config/ElideResourceConfig.java

```diff
@@ -58,8 +73,50 @@ protected void configure() {
             protected void configure() {
                 ElideSettings elideSettings = settings.getElideSettings(injector);
 
+                Elide elide = new Elide(elideSettings);
+                ObjectMapper mapper = elide.getMapper().getObjectMapper();
+                mapper.registerModule(
+                        new SimpleModule("isoDate", Version.unknownVersion())
+                                .addSerializer(Date.class, new JsonSerializer<Date>() {
+                                    @Override
+                                    public void serialize(Date date,
+                                                          JsonGenerator jsonGenerator,
+                                                          SerializerProvider serializerProvider)
+                                            throws IOException, JsonProcessingException {
+                                        if (date == null) {
+                                            jsonGenerator.writeNull();
+                                        } else {
+                                            DateFormat df = new SimpleDateFormat("yyyy-MM-dd'T'HH:mm'Z'");
+                                            // You can set timezone however you wish
+                                            df.setTimeZone(TimeZone.getDefault());
+                                            jsonGenerator.writeObject(df.format(date));
+                                        }
+                                    }
+                                })
+                );
+
+                CoerceUtil.register(new Converter() {
```

Comment:
Is this spelled out somewhere in the docs that a user may need to implement a converter? It’s unfortunate that we have to do this 2x (once for us and once for Jackson)

Reply:
Maybe we should wrap this in an interface we own? 

```java
interface Serializer<T> {
    public T convert(Class<T> aClass, Object o);
    public void serialize(T toWrite, JsonGenerator jsonGenerator, SerializerProvider serializerProvider);
} 
```

That way clients can just write the bits they care about and we can deal with registering it however we need to.